code/_onclick/hud/parallax.dm

```diff
@@ -6,6 +6,9 @@
 #define GRID_WIDTH 3
 
 var/list/parallax_on_clients = list()
+var/parallax_initialized = 0
+var/space_color = "#050505"
+var/list/parallax_icon[(GRID_WIDTH**2)*3]
```

Comment:
@clusterfack he wants it to be a define. I'll do the work for you
`#define THREE 3`
just copy that code in and replace the 3 with that define